### PR TITLE
Shell Sort

### PR DESCRIPTION
**Pull request by icoder18**
Adding Shell Sort to algorithms

### CODE REVIEW

**Comment by SrikanthYashaswi** [comment]
cool, but why have you closed it?

File: src/algorithms/shellSort.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const shellSort=(list)=>{
```
Comment (SrikanthYashaswi) [inline src/algorithms/shellSort.js:1]: export is missing..!

**Comment by icoder18** [comment]
I am fairly new to JS. 
I have corrected the file now 👍

**Comment by SrikanthYashaswi** [comment]
really cool, keep contributing..! 
you can suggest if any feature that can be brought to it..!

**Pull request merged**